### ISSUE TITLE
[craftercms] Warning message about problem with directory `CRAFTER_INSTALL/bin/apache-tomcat/shared/lib` when stopping crafter

### ISSUE THREAD

**Issue by alhambrav** [craftercms/craftercms#6024]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

### Describe the issue

There's a warning on the terminal when stopping crafter (both authoring and delivery)

```
WARNING: Problem with directory [/Users/vita/Desktop/crafter-auth/bin/apache-tomcat/shared/lib], exists: [false], isDirectory: [false], canRead: [false]
```

### Steps to reproduce

Steps:

1. Start crafter (./gradlew start)
2. Stop crafter (./gradlew stop)
3. Notice the warning message about problem with directory `CRAFTER_INSTALL/bin/apache-tomcat/shared/lib`

**Comment by sumerjabri** [comment]
Fixed in f5756953f56edd8e1c71a83cbb5c929db38300c1

**Comment by alhambrav** [comment]
Verified fixed
